### PR TITLE
Added UdpIdleTimeout flag

### PR DESCRIPTION
**Pull request by ArtfulCoder**

### CODE REVIEW

**Comment by thockin** [comment]
Needs rebase

File: cmd/kube-proxy/app/server.go [hunk 1]
```diff
@@ -92,6 +93,7 @@ func (s *ProxyServerConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.CleanupAndExit, "cleanup-iptables", false, "If true cleanup iptables rules and exit.")
 	fs.Float32Var(&s.KubeApiQps, "kube-api-qps", s.KubeApiQps, "QPS to use while talking with kubernetes apiserver")
 	fs.IntVar(&s.KubeApiBurst, "kube-api-burst", s.KubeApiBurst, "Burst to use while talking with kubernetes apiserver")
+	fs.DurationVar(&s.UdpIdleTimeout, "udp-timeout", s.UdpIdleTimeout, "How long an idle udp connection will be kept open (e.g. '250ms', '2s').  Must be greater than 0. Only applicable for userspace-proxier.")
```
Comment (thockin) [inline cmd/kube-proxy/app/server.go]: nits:

s/udp/UDP/

s/userspace-proxier/proxy-mode=userspace/

**Comment by thockin** [comment]
nits then LGTM

**Comment by ArtfulCoder** [comment]
Done. PTAL

**Comment by thockin** [comment]
needs rebase still

**Comment by eparis** [comment]
@lavalamp I thought we fixed the merge bot so he would just skip things that were not able to be merged, my dryrun is skipping this PR....

**Comment by eparis** [comment]
I'm going to clear LGTM, since this needs a rebase, as soon as you do rebase the bot will clear it.  That should also make the bot shut up.

File: cmd/kube-proxy/app/server.go [hunk 2]
```diff
@@ -66,6 +66,7 @@ type ProxyServerConfig struct {
 	CleanupAndExit     bool
 	KubeApiQps         float32
 	KubeApiBurst       int
+	UdpIdleTimeout     time.Duration
```
Comment (lavalamp) [inline cmd/kube-proxy/app/server.go]: While I'm here, this should be UDPIdleTimeout, please.

Comment (ArtfulCoder) [inline cmd/kube-proxy/app/server.go]: done

**Comment by ArtfulCoder** [comment]
Re-based
PTAL

**Comment by thockin** [comment]
needs gofmt.

**Comment by ArtfulCoder** [comment]
PTAL

**Comment by thockin** [comment]
LGTM - hope it goes in quickly :)

**Pull request merged**